packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/FilterStatus/FilterStatus.stories.js

```diff
@@ -0,0 +1,48 @@
+import zooTheme from '@zooniverse/grommet-theme'
+import { Grommet } from 'grommet'
+import React from 'react'
+
+import { task as mockTask } from '@plugins/tasks/SurveyTask/mock-data'
+
+import FilterStatus from './FilterStatus'
+
+function StoryContext (props) {
+  const { children, theme } = props
+
+  return (
+    <Grommet
+      background={{
+        dark: 'dark-1',
+        light: 'light-1'
+      }}
+      theme={theme}
+      themeMode={(theme.dark) ? 'dark' : 'light'}
+    >
+      {children}
+    </Grommet>
+  )
+}
+
+export default {
+  title: 'Tasks / SurveyTask / FilterStatus',
+  component: FilterStatus
+}
+
+const Template = ({
+  dark,
+  task
+}) => (
+  <StoryContext
+    theme={{ ...zooTheme, dark }}
+  >
+    <FilterStatus
```

Comment:
Maybe should wrap in a Box so DropContent is initially wider. If a filter is selected it widens, but currently the story is a little clunky.